src/charm.py

```diff
@@ -40,10 +50,13 @@ def __init__(self, *args):
         super().__init__(*args)
         self.workload = KyuubiServer(self.unit.get_container(KYUUBI_CONTAINER_NAME))
         self.s3_requirer = S3Requirer(self, S3_INTEGRATOR_REL)
+        self.database = DatabaseRequires(
```

Comment:
maybe we could call it `self.metastore`

Reply:
Changed to `self.metastore_db`. The other for the authentication will be `self.auth_db`.